docs/5.1. Transport - MQTT.md

```diff
@@ -80,4 +80,6 @@ This should be the `connection_lost` message described in [Message types - The c
 
 ## 7. Broker discovery
 
-The MQTT broker will be advertised via the DNS-SD service type `_nmos-mqtt._tcp`.
+The default MQTT broker should be advertised via the DNS-SD service type `_nmos-mqtt._tcp`.
+
+Each DNS-SD advertisement of this service type should be accompanied by a TXT record of name `api_proto` with a value of either `mqtt` or `secure-mqtt`, in order to indicate whether the service instance is using TLS on the port.
```

Comment:
Perhaps 'must' rather than 'should', otherwise there may need to be an assumed default.

Reply:
I dithered. Happy to put 'must', it's not the biggest change we'll have between v1.0 and v1.0.1.
@mjeras, does that sound OK?

Reply:
I've now made `api_proto` required.

Reply:
Agreed, that would be odd.